backend/src/commandTypes.ts

```diff
@@ -74,16 +72,16 @@ export const commandTypes = {
 
   async anyId(value: string, context: CommandContext<any>) {
     const userId = resolveUserId(context.pluginData.client, value);
```

Comment:
This function should return a Snowflake so you don't need to use type assertion after using it